docs/troubleshooting.md

```diff
@@ -0,0 +1,44 @@
+# Troubleshooting
+
+## Collecting logs from service binding manager
+
+1. Identify the `Pod` that is running the `service-binding-manager` container in the `service-bindings` namespace.
+
+    `$kubectl -n service-bindings get pods`
+
+    For example:
+    
+    ```bash
+    NAME                       READY   STATUS    RESTARTS   AGE
+    manager-5dbb5f584c-zvmg2   1/1     Running   0          11d
+    ```
+
+2. Gather the actual logs from the `Pod` using the `kubectl` command.
+   
+   `$ kubectl -n service-bindings logs manager-5dbb5f584c-zvmg2`
```

Comment:
Update kubectl command to use labels
`kubectl -n service-bindings logs -l role=manager` . This removes the need to get Pod name

Reply:
Thank you @shashwathi, good improvement